tests/test_tracking_grid/results_true.dat

```diff
@@ -0,0 +1,27 @@
+Diagonal track...
+Number of segments = 3
```

Comment:
Do we need the text metadata like "Segment", "length," "CMFD FWD=", etc. or can we do away with it and simply print numbers? For example, in place of:

```
Segment 0: length=2.82842707634, FSR ID=0, CMFD FWD=-1, CMFD BWD=-1, Material Name=Water, Material ID=10083
```

can we instead have:

```
2.82842707634, 0, 1, -1, Water, 10083
```

Of course this is a tradeoff between verbosity and bloating the size of our test suite and repo.